doc/source/examples/extended_examples/executable/executable.rst

```diff
@@ -101,6 +101,50 @@ This code shows how you can input other arguments:
    The first natural frequency is 344.28 Hz.
 
 
+Convert the app to an executable file
+=====================================
+
+Using the Python library `PyInstaller <https://pyinstaller.org>`_, you can convert the app to an executable file.
```

Comment:
Links should be in the `links.rst` file and not directly in the files: it is easier to maintain that way.